blog-posts/step-function-learnings/step-function-logs-learning.md

```diff
@@ -0,0 +1,92 @@
+---
+published: false
+title: "Stop looking for your express state machine logs, they don't exist"
+description: 'Logs for express Step Function with the CDK'
+tags: logs, stepfunction, serverless, cdk
+series:
+canonical_url:
+---
+
+In this article you will learn how to add **logs in an express step function** with the CDK because it is not enabled by default. When you go from a standard step function with logs to an express one that gives you nothing.
```

Comment:
Add `AWS step function` aswell.
The second sentence is a bit odd. You can reformulate with something along the line: `Since it is enabled by default on the Standard version, ...`
You should add multiple links to the documentation, ie. one for "it is not enabled by default" with this link: https://docs.aws.amazon.com/step-functions/latest/dg/diff-standard-express-exec-details-ui.html#exp-wf-exec-limitation-cw-reliance-test